src/views/virtualmachines/details/tabs/details/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem.tsx

```diff
@@ -29,6 +31,7 @@ type VirtualMachineDescriptionItemProps = {
   editOnTitleJustify?: boolean;
   label?: ReactNode;
   'data-test-id'?: string;
+  disabledEditTooltip?: string;
```

Comment:
This name sounds a bit like a boolean, like on off switch to disable. maybe replace the name to indicate its a message in the case of disabled?

Reply:
`messageOnDisabled` ?